src/basilisp/edn.lpy

```diff
@@ -88,7 +88,7 @@
   basilisp.lang.reader/ns-name-chars)
 
 (def ^:private num-chars
-  basilisp.lang.reader/num-chars)
+  #"[0-9]")
```

Comment:
`basilisp.lang.reader/num-chars` was removed because the main reader needs to support a much wider variety of integer literals than does [EDN](https://github.com/edn-format/edn).